### ISSUE TITLE
Fuff crashes if HTTP error code 300 received

### ISSUE THREAD

**Issue by MMquant** [ffuf/ffuf#296]
Fuff crashes if it gets 300 error code

for example consider following HTTP request/response which I got via curl
```
# cl 'http://<redacted>/.-0.htm'
HTTP/1.1 300 Multiple Choices
Date: Tue, 15 Sep 2020 14:31:35 GMT
Server: Apache/2.2.3 (CentOS)
Content-Length: 578
Connection: close
Content-Type: text/html; charset=iso-8859-1

<!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>300 Multiple Choices</title>
</head><body>
<h1>Multiple Choices</h1>
The document name you requested (<code>/.-0.htm</code>) could not be found on this server.
However, we found documents with names similar to the one you requested.<p>Available documents:
<ul>
<li><a href="/.well-known">/.well-known</a> (common basename)
<li><a href="/.">/.</a> (common basename)
<li><a href="/..">/..</a> (common basename)
</ul>
<hr>
<address>Apache/2.2.3 (CentOS) Server at <redacted> Port 80</address>
</body></html>
```

If fuff uses the `.-110` payload. It crashes and spits following stack trace

```
.-110                   [Status: 300, Size: 576, Words: 48, Lines: 16]
panic: runtime error: index out of range [0] with length 0

goroutine 1832 [running]:
github.com/ffuf/ffuf/pkg/ffuf.(*Response).GetRedirectLocation(0xc000255d70, 0xc000a99300, 0x832e78, 0x4)
	/root/go/src/github.com/ffuf/ffuf/pkg/ffuf/response.go:27 +0x17c
github.com/ffuf/ffuf/pkg/output.(*Stdoutput).Result(0xc00000e040, 0x12c, 0xc000a99290, 0xc000020600, 0x240, 0x600, 0x240, 0x30, 0x10, 0x0, ...)
	/root/go/src/github.com/ffuf/ffuf/pkg/output/stdout.go:236 +0x1c9
github.com/ffuf/ffuf/pkg/ffuf.(*Job).runTask(0xc0000121e0, 0xc000831380, 0x16d, 0xc000715900)
	/root/go/src/github.com/ffuf/ffuf/pkg/ffuf/job.go:303 +0x43e
github.com/ffuf/ffuf/pkg/ffuf.(*Job).startExecution.func1(0xc0001de000, 0xc0001beb60, 0xc0000121e0, 0xc000831380, 0x16d)
	/root/go/src/github.com/ffuf/ffuf/pkg/ffuf/job.go:159 +0xd0
created by github.com/ffuf/ffuf/pkg/ffuf.(*Job).startExecution
	/root/go/src/github.com/ffuf/ffuf/pkg/ffuf/job.go:156 +0x1a7
```

**Comment by joohoi** [comment]
Thanks for opening the issue, this is indeed a bug, but it was fixed in #265 so if you update to the latest version, you should be good!

I'm closing this as it has been already solved.

**Comment by MMquant** [comment]
My ffuf version is
```
# ffuf -h
Fuzz Faster U Fool - v1.1.0-git
```
which is official prebuilt latest release. I will try to compile ffuf from source.

**Comment by MMquant** [comment]
I confirm that the issue is fixed in `v1.2.0-git`.
